schema.md

```diff
@@ -0,0 +1,555 @@
+should be that of users of the software, not developers. You can think
+of this text as the description of the software that would be in its
+website (if the software had one).
+
+### Key `description/screenshots`
+
+* Type: array of strings (paths)
+* Presence: optional
+* Formats: PNG, JPG
+* Example: "data/screenshots/configuration.png"
+
+This key contains one or multiple paths to files showing screenshots of
+the software. They are meant to give a quick idea on how the software
+looks like and how it works.
+
+Screenshots can be of any shape and size; the suggested formats are:
+
+ * Desktop: 1280x800 @1x
+ * Tablet: 1024x768 @2x
+ * Mobile: 375x667 @2x
+
+
+### Key `description/videos`
+
+* Type: array of strings (URLs)
+* Presence: optional
+* Example: "https://youtube.com/xxxxxxxx"
+
+This key contains one or multiple URLs of videos showing how the
+software works. Like screenshots, videos should be used to give a quick
+overview on how the software looks like and how it works. Videos should
+be hosted on a video sharing website that supports the
+[oEmbed](https://oembed.com) standard; popular options are YouTube and
+Vimeo.
+
+
+
+
+
+
+
+
+
+
+
+## Section `meta`
+
+This section contains information useful to classify the software,
+define its scope of usage and its goal at a general level.
+
+### Key `meta/scope`
+
+* Type: string or array of strings
+* Presence: optional
+* Example: "es"
+
+This key defines the expected geographical scope of the software. If
+a software is probably useful only within some specific countries
+(because, eg., it helps implementing a specific regulation), specify
+the ISO693-1 two-letter country codes of the countries n this key.
+
+### Key `meta/pa-type`
+
+* Type: enumerated string or array of strings
+* Presence: optional
+* Values: see [pa-types.md](pa-types.md)
+* Example: "city"
+
+This key defines the types of public administration which is expected
+to use this software. Public software is often specific in scope, because
+there is a large set of tasks that are specific to each type of
+administration. For instance, many softwares that are used in schools
+are probably not useful in hospitals.
+
+The list of allowed values is defined in [pa-types.md](pa-types.md),
+and can evolve at any time, separately from the version of this
+specification.
+
+### Key `meta/category`
+
+* Type: enumerated string
+* Presence: optional
+* Values: see [sw-category.md](sw-category.md)
+* Example: "it-anagrafe"
+
+This key defines the "category" of software, which is the market area
+for the software. You can select the category that is more similar to
+the software; it doesn't have to be an exact match, but it can still
+be very useful for building catalogs of software and evaluate
+alternatives.
+
+The list of allowed values is defined in [sw-category.md](sw-category.md),
+and can evolve at any time, separately from the version of this
+specification.
+
+### Key `meta/tags`
```

Comment:
A bit more explanation on the difference between tags and categories would be useful.

Reply:
Correct. We're thinking of this. Our current reasoning is that "category" is one or more tags from a closed list, while "tags" is a list of open tags.